Tests/Unit/Hooks/PageRenderer/TrackingCodeInjectorTest.php

```diff
@@ -91,6 +101,21 @@ protected function setUp(): void
             ->willReturn($this->tagManagerCodeBuilderStub);
 
         $this->pageRendererMock = $this->createMock(PageRenderer::class);
+
+        $this->scriptTagBuilderStub = $this->createStub(ScriptTagBuilder::class);
+        $this->scriptTagBuilderStub
+            ->method('build')
+            ->willReturn('<script></script>');
+
+        $enrichEvent = new EnrichScriptTagEvent($this->requestStub);
+
+        $eventDispatcherStub = $this->createStub(EventDispatcherInterface::class);
+        $eventDispatcherStub
+            ->method('dispatch')
+            ->willReturn($enrichEvent);
+        // This builder will render a tag without additional parameters
+        $this->scriptTagBuilder = new ScriptTagBuilder($eventDispatcherStub);
```

Comment:
The concrete implementation is only used once. This way I would move it to the test method where it is needed. makes the test class cleaner, I think.

Reply:
It is in use more than once, that's why i did made a local version of it.

Reply:
If it's okay for you, please resolve this issue.